### PR TITLE
Update CodeSandbox Embed docs 

### PR DESCRIPTION
**Pull request by zacjones93**
##  What kind of change does this PR introduce?

docs

## What is the current behavior?

The screenshot and wording is a little out of date on the [Embed docs page](https://codesandbox.io/docs/embedding#embed-options). This updates the wording and screenshot to reflect the current UI.

1. Updated wording
2. Updated screenshot
3. Optimized screenshot image
4. Ran tests

## Checklist

- [x] Documentation
- [x] Testing 
- [x] Ready to be merged
- [x] Added myself to contributors table

### CODE REVIEW

File: packages/homepage/content/docs/2-embedding.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ the embed doesn't have all features of the full editor.
 
 ## Generate a URL
 
-You can generate a URL clicking 'Share' in the header and selecting the options
+You can generate a URL clicking 'Embed' in the header and selecting the options
 you want to have enabled.
 
 ![Share Button](./images/share-button.png)
```
Comment (MichaelDeBoey) [inline packages/homepage/content/docs/2-embedding.md]: Also rename the `.png` file

```suggestion
![Embed Button](./images/embed-button.png)
```

Comment (zacjones93) [reply packages/homepage/content/docs/2-embedding.md]: Makes sense!

**Comment by zacjones93** [comment]
Ah, my bad! Updated

**Comment by MichaelDeBoey** [comment]
Unfortunately @garethx has already changed this in the docs, so your PR is obsolete now.

**Pull request closed without merging**